tests/integration/test_complete_run.py

```diff
@@ -9,13 +9,18 @@
 # Copied from E3SM Diags
 def compare_images(
     test,
+    missing_images,
     mismatched_images,
     image_name,
     path_to_actual_png,
     path_to_expected_png,
 ):
     # https://stackoverflow.com/questions/35176639/compare-images-python-pil
-    actual_png = Image.open(path_to_actual_png).convert("RGB")
+    try:
+        actual_png = Image.open(path_to_actual_png).convert("RGB")
+    except FileNotFoundError:
+        missing_images.append(image_name)
```

Comment:
I added this because the test was erroring out because I changed the year-set and hadn't updated the expected files yet.